lib/spack/spack/util/package_hash.py

```diff
@@ -14,6 +14,8 @@
 import spack.util.naming
 from spack.util.unparse import unparse
 
+# FIXME: double check changes that are needed in this file
```

Comment:
Yes, they are needed because a few attributes are not always found. I need to double check if the solution to use `getattr` is fine (I think it is) or fi there are unexpected consequences to that. Maybe if @tgamblin could look at this file it would be good. If this seems ok I'll just remove the `FIXME:`

Reply:
Reverted.